shopfloor_mobile/static/wms/src/scenario/checkout.js

```diff
@@ -65,6 +65,7 @@ const Checkout = {
                     :select_records="state.data.picking.move_lines"
                     :select_records_grouped="utils.wms.group_lines_by_location(state.data.picking.move_lines, {'prepare_records': utils.wms.only_one_package})"
                     :select_options="select_line_manual_select_opts()"
+                    :use_qty_todo_by_pkg="true"
```

Comment:
as per select options I would encapsulate this setting for the qty picker into `:qty_picker_options`